internal/providers/terraform/azure/registry.go

```diff
@@ -236,11 +281,43 @@ var FreeResources []string = []string{
 	"azurerm_container_registry_token",
 	"azurerm_container_registry_webhook",
 
+	// Azure Sentinel
```

Comment:
Out of interest, any ideas on which TF resource we can/should show [these costs](https://azure.microsoft.com/en-us/pricing/details/azure-sentinel/ ) in?

Maybe we don't mark `azurerm_sentinel_data_connector_*` as free and instead show the `$2 per GB-ingested` price for each of them?

Reply:
To be in line with the DNS records, that would probably be a better idea to mark all the connectors with this pricing too. Good Suggestion.

Regarding the Azure Sentinel TF resource, the closest one would be `azurerm_log_analytics_workspace`.
The pricing on Sentinel is pretty weird because it is built on top of Log Analytics. 
If you enable Sentinel on a Log Analytics Workspace additional charges apply based on the amount of GB ingested. That means the standard rates for Log Analytics apply, and on top of that, you have to pay additionally the Azure Sentinel Charges per GB.

https://azure.com/e/8a92a412178147b081d19c771f43fca8

Reply:
Cool, I removed them from the free list in the master branch (I forgot to do it before merging this). I also added https://github.com/infracost/infracost/issues/889 for Azure Sentinel